File: DSA_LABS/PRACTICAL-1&2/Armstrong.java
public class Armstrong {
    public static void main(String[] args) {
        int size=10000;

        for(int i=1;i<=size;i++) {
            int n=i;
            int temp=n;
            int pow=0;
            while (temp != 0) {
                temp /= 10;
                pow++;
            }

            int sum=0;
            temp=n;

            while(temp != 0) {
                int digit=temp%10;
                sum += Math.pow(digit, pow);
                temp /= 10;
            }
            if (sum==n) {
                System.out.println(sum);
            }
        }
    }
}

File: Sem-3/WEB_TECH_LABS/LAB_7/Armstrong.js
let num=153;
let len=0;

let temp=num;
while(temp != 0){
    temp=Math.floor(temp/10);
    len++;
}

temp=num;
let sum=0;
for(let i=1;i<=len;i++){
    let digit=temp%10;
    temp=Math.floor(temp/10);
    sum=sum+Math.pow(digit,len);
}

if(num==sum)
    console.log(num+" is armstrong number.");
else
    console.log(num+" is not armstrong number.")
